src/classes/TDTM_DefaultConfig.cls

```diff
@@ -182,7 +182,7 @@ public without sharing class TDTM_DefaultConfig {
         // RecurringDonations on RecurringDonations
         handlers.add(new Trigger_Handler__c(Active__c = true, Asynchronous__c = false,
               Class__c = 'RD_RecurringDonations_TDTM', Load_Order__c = 1, Object__c = 'npe03__Recurring_Donation__c',
-              Trigger_Action__c = 'BeforeInsert;BeforeUpdate;AfterInsert;AfterUpdate'));
+              Trigger_Action__c = 'BeforeInsert;BeforeUpdate;AfterInsert;AfterUpdate;BeforeDelete'));
```

Comment:
The list of actions needs be in alphabetical order - only for the reason of how the logic that compares the defaults to the current records works.